src/main/java/edu/metrostate/ics499/team2/services/QuizService.java

```diff
@@ -72,6 +73,6 @@ public boolean isValid(Quiz obj) {
 		List<Quiz> result = quizRepo.findAll();
 				return result.stream()
 				.filter(quiz -> quiz.getQuestion().equalsIgnoreCase(obj.getQuestion()))
-				.filter(quiz -> quiz.getAnswer().equalsIgnoreCase(obj.getAnswer())).toList().size() > 0 ? false : true;
+				.filter(quiz -> quiz.getAnswer().equalsIgnoreCase(obj.getAnswer())).collect(Collectors.toList()).size() > 0 ? false : true;
```

Comment:
the issue with this in IntelliJ was `toList()` outside of the `collect()` only works on Java 16+.